baseplate/events/queue.py

```diff
@@ -170,6 +171,19 @@ def serialize_v2_event(event):
     return Serializer.serialize(_V2_PROTOCOL_FACTORY, event)
 
 
+class EventLogger(object):
+    def log(self, **kwargs):
+        raise NotImplementedError
+
+
+class DebugLogger(EventLogger):
```

Comment:
might be nice to call this `DebugEventLogger`, sorry for missing that in last pass

Reply:
Makes sense. Will fix.